package.json

```
@@ -34,45 +32,35 @@
     "stat-mode": "0.3.0",
     "stream-to-promise": "2.2.0",
     "tar": "4.4.18",
+    "tinyexec": "0.3.2",
     "tree-kill": "1.2.2",
     "uid-promise": "1.0.0",
     "uuid": "3.3.2",
     "xdg-app-paths": "5.1.0",
     "yauzl-promise": "2.1.3"
   },
   "devDependencies": {
+    "@esbuild/darwin-arm64": "^0.24.2",
```

```suggestion
    "@esbuild/darwin-arm64": "0.24.2",
```
Pin dependency to specific version.